app/views/todos/_form.html.erb

```diff
@@ -1,25 +0,0 @@
-<%= form_for(@todo) do |f| %>
```

Comment:
[Nits]
you forgot to remove `models/to_do.rb` :sweat_smile:

Reply:
i remove it should show background color red